pfcpiface/fastpath.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 type fastPath interface {
+	/* Close any pending sessions */
 	exit()
+	/* setup internal parameters and channel with fastPath */
 	setUpfInfo(u *upf, conf *Conf)
+	/* set udp and pfcp conn parameters in interface */
 	setInfo(udpConn *net.UDPConn, udpAddr net.Addr, pconn *PFCPConn)
+	/* simulator mode setup */
 	sim(u *upf, method string)
-	sendMsgToUPF(method string, pdrs []pdr, fars []far) uint8
-	sendDeleteAllSessionsMsgtoUPF()
+	/* write pdr/far/qer to fastpath */
+	sendMsgToUPF(u *upf, method string, pdrs []pdr, fars []far) uint8
```

Comment:
why does it need access to upf everytime? idea of setupinfo was to setup everything before hand internal to specific fastpath